spark-indexer/src/test/java/org/uniprot/store/spark/indexer/proteome/ProteomeDocumentsToHPSWriterIT.java

```diff
@@ -0,0 +1,624 @@
+                                                                        .database(
+                                                                                CitationDatabase
+                                                                                        .DOI)
+                                                                        .id("10.1128/jvi.00637-11")
+                                                                        .build())))
+                                        .journalName("J. Virol.")
+                                        .build())),
+                proteomeEntry.getCitations());
+    }
+
+    private void assertProteomeEntry2(ProteomeEntry proteomeEntry) {
+        assertEquals(
+                new GenomeAnnotationBuilder()
+                        .source("ENA/EMBL")
+                        .url("https://www.ebi.ac.uk/ena/browser/view/GCA_003971765.1")
+                        .build(),
+                proteomeEntry.getGenomeAnnotation());
+        assertEquals(
+                new ArrayList<>(
+                        List.of(
+                                new ComponentBuilder()
+                                        .name("Genome")
+                                        .description("Lelystad virus")
+                                        .genomeAnnotation(
+                                                new GenomeAnnotationBuilder()
+                                                        .source("ENA/EMBL")
+                                                        .build())
+                                        .proteomeCrossReferencesSet(
+                                                new ArrayList<>(
+                                                        List.of(
+                                                                new CrossReferenceBuilder<
+                                                                                ProteomeDatabase>()
+                                                                        .database(
+                                                                                ProteomeDatabase
+                                                                                        .GENOME_ACCESSION)
+                                                                        .id("M96262")
+                                                                        .build())))
+                                        .proteinCount(11)
+                                        .build())),
+                proteomeEntry.getComponents());
+        assertEquals(
+                new ProteomeStatisticsBuilder()
+                        .reviewedProteinCount(1)
+                        .unreviewedProteinCount(0)
+                        .isoformProteinCount(0)
+                        .build(),
+                proteomeEntry.getProteomeStatistics());
+    }
+
+    private void assertProteomeEntry3(ProteomeEntry proteomeEntry) {}
```

Comment:
assertProteomeEntry3, assertProteomeEntry4, assertProteomeEntry5 and assertProteomeEntry6 are empty, can you please implement it?